readme.md

```diff
@@ -220,4 +220,6 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** Where do computers go to dance?
 
+**A:** The disk-o. They really love techno music.
```

Comment:
I feel like this is two jokes.

Where do they go to dance? disk-o

What music do they listen to? techno

But neither one really tickles my funny bone 🤷‍♂️